packages/lesswrong/components/hooks/useInitiateConversation.ts

```diff
@@ -33,6 +33,7 @@ export const useInitiateConversation = (props?: { includeModerators?: boolean })
     },
     collectionName: "Conversations",
     fragmentName: "ConversationsMinimumInfo",
+    fetchPolicy: "network-only",
```

Comment:
Fix for a bug where, if you add a title to a conversation and then immediately try to create a new one with the same participants, it fetched the _titled_ conversation rather than creating a new untitled one.